### PR TITLE
docs(line-ripple): Fix typos and add hyphen to improve readability

### PR DESCRIPTION
**Pull request by kfranqueiro**
Also fixes inconsistencies between things ending with a period or not in tables.

### CODE REVIEW

File: packages/mdc-line-ripple/README.md [hunk 1]
```diff
@@ -42,15 +42,15 @@ CSS Class | Description
...
+`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple
 
 ### Sass Mixins
 
 ### `MDCLineRipple`
 
-`activate() => void` | Proxies to the foundations `activate()` method.
-`deactivate() => void` |  Proxies to the foundations `deactivate()` method.
-`setRippleCenter(number: xCoordinate) => void` | Proxies to the foundations `setRippleCenter(number: xCoordinate)` method.
+`activate() => void` | Proxies to the foundation's `activate()` method
```
Comment (moog16) [inline packages/mdc-line-ripple/README.md]: Is this supposed to be a table? I think you need the `--- | ---` above

Comment (kfranqueiro) [reply packages/mdc-line-ripple/README.md]: You, sir, are very correct, and wowzers this sure looks very broken right now. Good catch.

File: packages/mdc-line-ripple/README.md [hunk 2]
```diff
@@ -42,15 +42,15 @@ CSS Class | Description
 --- | ---
 `mdc-line-ripple` | Mandatory
 `mdc-line-ripple--active` | Styles the line ripple as an active line ripple
-`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple. 
+`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple
 
 ### Sass Mixins
```
Comment (moog16) [inline packages/mdc-line-ripple/README.md]: I think this is unnecessary

Comment (kfranqueiro) [reply packages/mdc-line-ripple/README.md]: Hm. Actually it looks like there _is_ a public mixin but it was never documented. Guess I'll add that in here.

**Pull request merged**